tests/unit_tests/CMakeLists.txt

```diff
@@ -7,7 +7,8 @@
 # Set up all of the "host" tests.
 add_subdirectory( core )
 add_subdirectory( io )
-add_subdirectory ( cpu )
+add_subdirectory( utils )
```

Comment:
We should also move the simulation test here, I guess

Reply:
Indeed